docs/dependencies.md

```diff
@@ -107,6 +109,10 @@ module.exports = {
 
 See [`script_phase` options](https://www.rubydoc.info/gems/cocoapods-core/Pod/Podfile/DSL#script_phase-instance_method) for a full list of available object keys.
 
+#### platforms.ios.configurations
+
+An array of build configurations which will include the dependency. If the array is empty, your dependency will be installed in all configurations. If you're working on a helper library, you should set debug as a value, to avoid shipping it as a part of the release process. For more details, see [`build configurations`](https://guides.cocoapods.org/syntax/podfile.html#pod).
```

Comment:
```suggestion
An array of build configurations which will include the dependency. If the array is empty, your dependency will be installed in all configurations. If you're working on a helper library that should only be included in development, such as a replacement for the React Native development menu, you should set this to `['debug']` to avoid shipping the library in a release build. For more details, see [`build configurations`](https://guides.cocoapods.org/syntax/podfile.html#pod).
```